gdx/src/com/badlogic/gdx/maps/tiled/renderers/IsometricTiledMapRenderer.java

```diff
@@ -40,26 +40,44 @@
 	private Vector2 topLeft = new Vector2();
 	private Vector2 bottomRight = new Vector2();
 
+	private float unitScaleY;
```

Comment:
What's with https://github.com/del-sol/libgdx/blob/master/gdx/src/com/badlogic/gdx/maps/tiled/renderers/BatchTiledMapRenderer.java#L197-L200 and https://github.com/del-sol/libgdx/blob/master/gdx/src/com/badlogic/gdx/maps/tiled/renderers/IsometricTiledMapRenderer.java#L145-L148 ?